Cluster-Backup/Kasten-Minikube/Mac-delete.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export NAMESPACE=kasten-io
+
+
+#Delete all resources in the namespace
+echo "--------------------Delete All Resources--------------------"
+kubectl delete namespace NAMESPACE --wait=false
```

Comment:
this command and the next one doing almost the same thing but I recommend using the second command only and removing this one